### ISSUE TITLE
There is a case that the loss becomes to Nan value

### ISSUE THREAD

**Issue by rose-jinyang** [wangchi95/active-boundary-loss#3]
Hello
How are you?
Thanks for contributing to this project.
Have u ever tried to train a model with ONLY this loss?
In the case, I found a case that this loss becomes to Nan value.

**Comment by wangchi95** [comment]
ABL is used to supervise the boundary alignment rather than the classification accuracy. Thus, another loss (like CE) that ensures classification accuracy is necessary.

In our experiments, CE+lovász-softmax loss+ABL was the best combination, as the quantitative results of the paper show. 
And the reason for introducing the lovász-softmax loss can be found in the last paragraph of Chapter _Training Loss_ in the paper.

**Comment by rose-jinyang** [comment]
Thanks
